cli-plugins/insight/cli-overview.hbs.md

```diff
@@ -6,9 +6,16 @@ Follow the below steps to install, configure, and use the Tanzu Insight CLI plug
 
 >**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform Build profile or Tanzu Application Platform View profile.
```

Comment:
Sorry! I happened to look up here by happenstance. 

The store being in `Tanzu Application Platform Build profile` is no longer true. We're only in the View profile now (as far as I know anyway).